docker-compose.yaml

```diff
@@ -4,17 +4,17 @@ services:
     build:
       context: .
       dockerfile: ./dockerfiles/dev.Dockerfile
-    working_dir: /go/src/github.com/ismaelchess/go-share-secret
+    working_dir: ${WORKING_DIR}
     volumes: 
-      - .:/go/src/github.com/ismaelchess/go-share-secret
+      - .:${WORKING_DIR}
     ports: 
       - ${PORT}:${PORT}
 
   redis:
     image: redis:5-alpine
     command: "redis-server --appendonly yes"
     volumes:
-      - /home/ismael/go/src/github.com/ismaelchess/redis/data:/data
+      - ${HOME}/go/redis/data:/data
```

Comment:
Not a good idea use `go` directory for storing redis data. Again, this should come from environment variables
https://docs.docker.com/compose/environment-variables/